packages/preview/fontawesome/0.5.0/README.md

```diff
@@ -0,0 +1,162 @@
+
+## Usage
+
+### Install the fonts
+
+You can download the fonts from the official website: https://fontawesome.com/download
+
+After downloading the zip file, you can install the fonts depending on your OS.
+
+#### Typst web app
+
+You can simply upload the `otf` files to the web app and use them with this package.
+
+#### Mac
+
+You can double click the `otf` files to install them.
+
+#### Windows
+
+You can right-click the `otf` files and select `Install`.
+
+#### Some notes
+
+This library is tested with the otf files of the Font Awesome Free set. TrueType fonts may not work as expected. (Though I am not sure whether Font Awesome provides TrueType fonts, some issue is reported with TrueType fonts.)
+
+### Import the library
+
+#### Using the typst packages
+
+You can install the library using the typst packages:
+
+`#import "@preview/fontawesome:0.5.0": *`
+
+#### Manually install
+
+Copy all files start with `lib` to your project and import the library:
+
+`#import "lib.typ": *`
+
+There are three files:
+
+- `lib.typ`: The main entrypoint of the library.
+- `lib-impl.typ`: The implementation of `fa-icon`.
+- `lib-gen.typ`: The generated icon map and functions.
+
+I recommend renaming these files to avoid conflicts with other libraries.
+
+### Use the icons
+
+You can use the `fa-icon` function to create an icon with its name:
+
+`#fa-icon("chess-queen")`
+
+Or you can use the `fa-` prefix to create an icon with its name:
+
+`#fa-chess-queen()` (This is equivalent to `#fa-icon().with("chess-queen")`)
+
+You can also set `solid` to `true` to use the solid version of the icon:
+
+`#fa-icon("chess-queen", solid: true)`
+
+Some icons only have the solid version in the Free set, so you need to set `solid` to `true` to use them if you are using the Free set.
+Otherwise, you may not get the expected glyph.
+
+#### Full list of icons
+
+You can find all icons on the [official website](https://fontawesome.com/search)
+
+#### Different sets
+
+By default, the library supports `Free`, `Brands`, `Pro`, `Duotone` and `Sharp` sets.
+(See [Enable Pro sets](#enable-pro-sets) for enabling Pro sets.)
+
+But only `Free` and `Brands` are tested by me.
+That is, three font files are used to test:
+
+- Font Awesome 6 Free (Also named as _Font Awesome 6 Free Regular_)
+- Font Awesome 6 Free Solid
+- Font Awesome 6 Brands
+
+Due to some limitations of typst 0.12.0, the regular and solid versions are treated as different fonts.
+In this library, `solid` is used to switch between the regular and solid versions.
+
+To use other sets or specify one set, you can pass the `font` parameter to the inner `text` function: \
+`fa-icon("github", font: "Font Awesome 6 Pro Solid")`
+
+If you have Font Awesome Pro, please help me test the library with the Pro set.
+Any feedback is appreciated.
+
+##### Enable Pro sets
+
+Typst 0.12.0 raise a warning when the font is not found.
+To use the Pro set, `typst #fa_use_pro()` should be called before any `fa-*` functions.
```

Comment:
If you want to have Typst syntax highlighting, I think the correct syntax is with three backticks. Also, the function should be renamed here too.

Reply:
Oh, this is a copy-paste mistake.

Thanks for helping to find the remaining unchanged function names. A commit will be pushed ASAP